swagger.yaml

```diff
@@ -1573,3 +1592,162 @@ definitions:
       node_id:
         description: "The id of the node"
         type: string
+  MetaData:
+    type: object
+    description: "An object containing all meta data information against a version"
+    properties:
+      alerts:
+        type: array
+        description: "A list of alerts, for example corrections after the resource has been published"
+        items:
+          $ref: '#/definitions/Alert'
+      collection_id:
```

Comment:
Do we currently make the collection_id public, or is it only available on auth'd requests? We don't want to be making this detail public, and i cant think of anywhere in our current usecase that we'd need this from this endpoint anyway

Reply:
It is publicly available on the dataset and version endpoints.

Reply:
OK, will raise something in the future to fix that - probably not what we want long term but certainly not the responsibility of this PR to fix if it already exists